src/auth/user-record.ts

```diff
@@ -56,6 +56,12 @@ export interface TotpInfoResponse {
   [key: string]: unknown;
 }
 
+export interface PasskeyInfoResponse {
+  name?: string;
+  credentialId?: string;
```

Comment:
credentialId and name should never be empty correct?

Reply:
yes